.github/CHANGELOG.md

```diff
@@ -48,6 +48,10 @@
 * Changed the name of `lightning.tensor` to `default.tensor` with the `quimb` backend.
   [(#719)](https://github.com/PennyLaneAI/pennylane-lightning/pull/719)
 
+* Patch the C++ `Measurements.probs(wires)` method in Lightning-Qubit and Lighnting-Kokkos to `Measurements.probs()` when called with all wires.
```

Comment:
What about Lightning-GPU?

Reply:
The `probs()` method in LGPU calls `probs(wires)` following the cuTensorNet API. This patch is not useful there. 

https://github.com/PennyLaneAI/pennylane-lightning/blob/81870b7bc427ce2362e29a64d77f33aa80062104/pennylane_lightning/core/src/simulators/lightning_gpu/measurements/MeasurementsGPU.hpp#L156-L162